### PR TITLE
Add option to provide a standalone script URL for running standalone …

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
## Description

- Add option to provide a standalone script URL for running standalone tool from any repository or branch using environment variable.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: tests/ilab/ilab_dw_rhoai_test.go [hunk 1]
```diff
@@ -33,8 +33,9 @@ import (
 
 // setting some defaults in case not provided.
 const (
-	ILAB_RHELAI_WORKBENCH_IMAGE = "quay.io/opendatahub/workbench-images:jupyter-datascience-ubi9-python-3.11-20241004-609ffb8"
-	ILAB_RHELAI_STORAGE_CLASS   = "nfs-csi"
+	ILAB_RHELAI_WORKBENCH_IMAGE   = "quay.io/opendatahub/workbench-images:jupyter-datascience-ubi9-python-3.11-20241004-609ffb8"
+	ILAB_RHELAI_STORAGE_CLASS     = "nfs-csi"
+	DEFAULT_STANDALONE_SCRIPT_URL = "https://raw.githubusercontent.com/opendatahub-io/ilab-on-ocp/refs/heads/phase-1/standalone/standalone.py"
```
Comment (sutaakar) [inline tests/ilab/ilab_dw_rhoai_test.go:38]: Personally I would rather throw out default script location and enforce user to specify script to be tested.
That way user is always aware of what script is being used.

Comment (abhijeet-dhumal) [reply tests/ilab/ilab_dw_rhoai_test.go:38]: Yes, But using default repo/branch script for testing will be always preferred right?

For example : 
The standalone script from this same repo from phase-1 branch

Comment (sutaakar) [reply tests/ilab/ilab_dw_rhoai_test.go:38]: Yes, but this test is not in the same repo or branch :)

Comment (abhijeet-dhumal) [reply tests/ilab/ilab_dw_rhoai_test.go]: Yes my mistake, my initial thought was the standalone script will be present in dw as well as in ilab-on-ocp repo, but it is removed now so keeping approach to provide standalone script URL mandatorily while running test..

Thanks @sutaakar for reviewing this PR !!

**Pull request closed without merging**